src/pinning.jl

```diff
@@ -137,6 +140,7 @@ Per default, a reasonable value for `places` is chosen based on the given `pinni
 * `:spread` or `scatter`: pins to `places` in an alternating / round robin fashion.
 * `:random`: shuffles the given `places` and then pins to them compactly.
 * `:current`: pins threads to the cpu threads where they are currently running (ignores `places`).
+* `:firstn`: pin threads to the cpu ids `0:nthreads - 1`.
```

Comment:
This is only correct if the first n cpu ids are indeed 0,1,2,3,... etc. which isn't the case on all systems. Instead, `:firstn` pins to the first n cpuids (as reported by lscpu, which is the basis for everything in ThreadPinning.jl).

Reply:
Right, I keep forgetting that :facepalm: Fixed in e9b3810.